lib/src/password-input/PasswordInput.test.js

```diff
@@ -79,15 +75,14 @@ describe("Password input component tests", () => {
     expect(queryByTitle("Hide password")).toBeFalsy();
   });
 
-  test("Password input has correct accesibility attributes", () => {
+  test("Password input has correct accessibility attributes", () => {
     const { getByRole, getByLabelText } = render(<DxcPasswordInput label="Password input" />);
     const passwordInput = getByLabelText("Password input");
-    expect(passwordInput.getAttribute("aria-autocomplete")).toBeNull();
-    expect(passwordInput.getAttribute("aria-controls")).toBeNull();
-    expect(passwordInput.getAttribute("aria-expanded")).toBeNull();
     const showButton = getByRole("button");
+    expect(passwordInput.getAttribute("aria-expanded")).toBe("false");
     expect(showButton.getAttribute("aria-label")).toBe("Show password");
     userEvent.click(showButton);
+    expect(passwordInput.getAttribute("aria-expanded")).toBe("true");
```

Comment:
I remove the checks of the `aria-autocomplete`, `aria-controls` since they were part of the behaviour of the regular Text Input and they are checked in its respective tests. But, I don't have any problem in restoring them. 

In regards to the `aria-expanded`, I think its value should only be true or false, depending on the visibility of the password text. So, I propose to keep it as it is.

Reply:
Agree!